modules/ingestor/tests/reingest/license.rs

```diff
@@ -0,0 +1,73 @@
+#![allow(clippy::expect_used)]
```

Comment:
This is not a "license" feature, please change the name of the module into something more appropriate.